plugin_test.go

```diff
@@ -37,31 +37,55 @@ func TestPluginExecuteCommand(t *testing.T) {
 }
 
 func TestPluginExecuteCommandHelp(t *testing.T) {
-	assert := assert.New(t)
 	p := &main.MatterpollPlugin{}
 
 	r, err := p.ExecuteCommand(&model.CommandArgs{
 		Command: `/matterpoll`,
 	})
 
-	assert.Nil(err)
+	assert.Nil(t, err)
+	isHelpResponce(t, r)
+}
+
+func TestPluginExecuteOneArgument(t *testing.T) {
+	p := &main.MatterpollPlugin{}
+
+	r, err := p.ExecuteCommand(&model.CommandArgs{
+		Command: `/matterpoll "abcd"`,
+	})
+	assert.Nil(t, err)
+	isHelpResponce(t, r)
+}
+
+func TestPluginExecuteTwoArguments(t *testing.T) {
+	p := &main.MatterpollPlugin{}
+
+	r, err := p.ExecuteCommand(&model.CommandArgs{
+		Command: `/matterpoll "abcd" "abcd"`,
+	})
+	assert.Nil(t, err)
+	isHelpResponce(t, r)
+}
+
+func isHelpResponce(t *testing.T, r *model.CommandResponse) {
```

Comment:
This assert function name should be `assertHelpResponse` since I think that functions started with `is`  or `has` should return true/false.

Reply:
:+1: